rootfs/etc/profile.d/prompt.sh

```diff
@@ -28,12 +28,25 @@ function terraform_prompt() {
 # Define our own prompt
 PROMPT_HOOKS+=("geodesic_prompt")
 function geodesic_prompt() {
+       
+  case $PROMPT_STYLE in
+      plain)
+          # 8859-1 codepoints:
+          WHITE_HEAVY_CHECK_MARK=$(tput bold)$(tput setab 2)$'X'$(tput sgr0)' '
+          BLACK_RIGHTWARDS_ARROWHEAD=$'=> '
+          TWO_JOINED_SQUARES=$'¤ '  # perhaps § instead?
+          CROSS_MARK=$'× '
+          ;;
+      *)
+          # unicode
+          WHITE_HEAVY_CHECK_MARK=$'\u2714 '     # '✔'
+          BLACK_RIGHTWARDS_ARROWHEAD=$'\u27A4 ' # '➤', suggest '▶' may be present in more fonts
+          TWO_JOINED_SQUARES=$'\u29C9 '         # '⧉'
+          CROSS_MARK=$'\u274C '                 # '❌'
+          ;;
+  esac
 
-  WHITE_HEAVY_CHECK_MARK=$'\u2705 '
-  BLACK_RIGHTWARDS_ARROWHEAD=$'\u27A4 '
-  TWO_JOINED_SQUARES=$'\u29C9 '
-  CROSS_MARK=$'\u274C '
-
+  # why do we 'export' STATUS?
```

Comment:
This might be legacy. Don't think the `export` is needed.

Reply:
if it works without it, drop it.

Reply:
dropped.